packages/frontend/src/utils/firstAndLast.ts

```diff
@@ -0,0 +1,10 @@
+export const firstAndLast = (value: string, first = 4, last = 4, middle = '...') => {
+  if (value.length < first + last) {
```

Comment:
This should be `value.length <= first + last` as it will now place dots in the middle. Passing in 12345678 would return 1234...5678

Reply:
We should add unit tests for this function in a follow-up PR too.

Reply:
Addressed in 20ff4deaf1cf18b8d1da4b594ac9b5a62c3f8a32.